hack/scripts/provision.sh

```diff
@@ -754,6 +776,10 @@ cmd_all() {
 		"--dev")
 			DEVELOPMENT=true
 			;;
+		"--auth")
+		  shift
```

Comment:
Why is this is indented this weirdly?

Reply:
🤷‍♀️ did you run a shell formatter over it? that should even everything out

Reply:
My IDE is crap. :( It didn't insert the right indentation character. I do have shell checker, but not formatter, I think. I shall install it. :)